data/json/items/tool/electronics.json

```
@@ -426,6 +426,46 @@
     "use_action": [ "NOISE_EMITTER_ON" ],
     "flags": [ "RADIO_MODABLE", "RADIO_INVOKE_PROC", "TRADER_AVOID", "WATER_BREAK" ]
   },
+  {
+    "id": "emf_detector",
+    "//": "Based off the TRIFIELD EMF Meter Model TF2",
+    "type": "TOOL",
+    "name": { "str": "EMF detector (off)", "str_pl": "EMF detectors (off)" },
+    "description": "This device detects and reads out different wave lengths.  It beeps with proximity and is sensitive enough you could use it to get a sense of direction.",
```

```suggestion
    "description": "This device detects and reads out different electromagnetic wave lengths.  It beeps with proximity and is sensitive enough you could use it to get a sense of direction.",
```

Something like that, otherwise one might confuse it with _any_ kinds of waves, like water waves.